virtme/guest/virtme-sshd-script

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Initialize ssh server for remote connections (option `--server ssh`)
+
+# Generate authorized_keys adding the host's user public keys.
+HOME=$(getent passwd "${ssh_user}" | cut -d: -f6)
+cat ${HOME}/.ssh/id_*.pub >> ${HOME}/.ssh/authorized_keys
```

Comment:
(do you think we should add double quotes around `HOME` just to be on the safe side? same below)